cmd/main.go

```diff
@@ -104,6 +104,7 @@ var Options struct {
 	LogConfig                   logconfig.Config
 	LeaderConfig                leader.Config
 	DeletionWorkerInterval      time.Duration `envconfig:"DELETION_WORKER_INTERVAL" default:"1h"`
+	DeregisterWorkerInterval    time.Duration `envconfig:"DEREGISTER_WORKER_INTERVAL" default:"1h"`
```

Comment:
please rename it to DeregisterInactiiveClustersInterval

Reply:
because it has meaning now and I don't understand why we need "Worker" inside the name